postgres-fdw/README.md

```diff
@@ -0,0 +1,119 @@
+# GreptimeDB for Postgres_fdw
+
+This docker-compose file demos how to configure GreptimeDB with Postgres foreign
+data wrapper so that you can query GreptimeDB from vanilla Postgres.
+
+This demo uses [Vector](https://vector.dev) as a data source to generate demo
+logs and ingest into GreptimeDB with built-in GreptimeDB sink.
+
+## How to run this demo
+
+Ensure you have `git`, `docker`, `docker-compose` and `psql` client
+installed. Docker Compose version 2.24 or higher is required. To run this
+demo:
+
+```shell
+git clone https://github.com/GreptimeTeam/demo-scene.git
+cd demo-scene/postgres-fdw
+docker compose up
+```
+
+You can access GreptimeDB using `psql` client. Just run `psql -h 127.0.0.1 -p
+4003 -d public` to connect to the database and use SQL query like `SHOW TABLES`
+as a start.
+
+```
+psql -h 127.0.0.1 -p 4003 -d public
+psql (16.5, server 16.3-greptimedb-0.11.0)
+Type "help" for help.
+
+public=> show tables;
+     Tables
+----------------
+ demo_logs_json
+ numbers
+(2 rows)
+```
+
+Next, use `psql` to access vanilla postgres using `psql -h 127.0.0.1 -p 5432 -U
+postgres`. You can check foreign servers and tables, query data from the remote
+GreptimeDB.
```

Comment:
Do we need to explain briefly how PostgreSQL recognizes GreptimeDB? Maybe a link to `init.sh`

Reply:
It's explained in "How it works" section below.